seed/lib/superperms/orgs/migrations/0018_update_organization_display_meter_units.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.10 on 2020-03-27 19:13
+
+from django.db import migrations
+
+
+def update_display_meter_units_options(apps, schema_editor):
+    Organization = apps.get_model('orgs', 'Organization')
+    for org in Organization.objects.all():
+        org.display_meter_units['Electric - Unknown'] = 'kWh (thousand Watt-hours)'
+        org.save()
```

Comment:
not very efficient, but I didn't want to spend too much time trying to figure out the SQL to make this query happen

Reply:
IMO it's probably fine - the magnitude of orgs in production is in the 100s, probably closer to 100 than 1000 even.